app/models/provider.rb

```diff
@@ -5,6 +5,7 @@ class Provider < ApplicationRecord
   include EmsRefresh::Manager
   include SupportsFeatureMixin
   include TenancyMixin
+  include UuidMixin
```

Comment:
@d-m-u I think this will require at least a data migration to add guids for existing records

Reply:
https://github.com/ManageIQ/manageiq-schema/pull/341/files#diff-286d3f057a70979e9298db00b23c6356R1

Reply:
It was added here: https://github.com/ManageIQ/manageiq-schema/pull/341/files#diff-286d3f057a70979e9298db00b23c6356R9